code/edge/src/androidTest/java/com/adobe/marketing/mobile/ConfigOverridesFunctionalTests.kt

```diff
@@ -431,15 +376,21 @@ class ConfigOverridesFunctionalTests {
         assertEquals(1, resultRequests.size.toLong())
 
         // Assert that default datastream ID value is in the URL query param for configId
-        assertEquals(CONFIG_ID, resultRequests[0]!!.queryParam("configId"))
+        assertEquals(CONFIG_ID, TestableNetworkRequest(resultRequests[0]).queryParam("configId"))
```

Comment:
I see that we are using TestableNetworkRequest to avoid forced unwrapping but TestableNetworkRequest doesn't have null checks and if the request is null we will see a crash. So we need to unwrap it.

```
assertEquals(CONFIG_ID, resultRequests[0]?.queryParam("configId") ?: "null"))
```